courses/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+# Create your models here.
+
```

Comment:
add a meta RowInformation column for most of the tables. Use your discretion wherever necessary. Should contain `Is_active`, `created_at` and `modified_at`